etc/kayobe/kolla.yml

```diff
@@ -12,11 +12,15 @@
 
 # URL of Kolla source code repository if type is 'source'.
 #kolla_source_url:
+kolla_source_url: https://git.openstack.org/openstack/kolla
 
 # Version (branch, tag, etc.) of Kolla source code repository if type is
 # 'source'.
 #kolla_source_version:
 
+# Uncomment for building magnum/6.0.0.0 image
```

Comment:
I think you can override it on the command line with ```-e kolla_source_version=stable/queens```. It's probably best to do it that way. Then you can have a one liner for building the queens magnum image, and not have to bother uncommenting stuff in here. It also helps keep the config clean.

Reply:
Nice point. Kayobe power user!

Reply:
I'm looking forward to my ewok sticker.